### PR TITLE
Nukie med bundle now costs 24 tc and contains a unique defibrillator

### PR DESCRIPTION
**Pull request by Blackern5000**
## About the PR
The nukie medical bundle now costs 24 tc but has received a huge buff to its contents.
For 24 tc you get...

image
- x1 Interdyne Defib
- x4 combat kit
  - x40 suture
  - x40 mesh
  - x4 brute injector
    - 15u bicaridine
    - 5u tran acid
  - x4 burn injector
    - 10u dermaline
    - 10u leporazine
  - x4 ephedrine syringe
  - x4 saline syringe
- x4 tourniquet
- x4 punct injector
  - 10u puncturase
  - 5u tran acid
- x4 pyra injector
  - 10u pyrazine
  - 10u dermaline
- x4 airloss injector
  - 20u saline
  - 20u dex+
- x4 combat medipen 
  - 25u omnizine
  - 5u tran acid
  
In total for chemicals in autoinjectors...
- 60u bicaridine
- 80u dermaline
- 40u leporazine
- 40u puncturase
- 40u pyrazine
- 80u saline
- 80u dex+
- 100u omnizine
- 60u tran acid

The interdyne defibrillator is a black and red one-handed defibrillator that comes with a medium power cell. It deals 8 blunt damage a hit, but when turned on does an additional 16 shock damage, for a total of 24 DPS. not equal to an esword or northstars, but still pretty good in a pinch.
The downside is that it takes 6 seconds to shock someone instead of 3, meaning you can be much more easily interrupted if you try to revive an ally in an unsafe environment.
The shock damage is not blocked by insulated gloves... Or any armor. It's effectively an AP melee weapon since shock resistance doesn't really exist. I'm willing to change it to just deal heat damage instead if this is too big of an issue.

## Why / Balance
The medical bundle has been more or less worthless for awhile now, and ever since a defib was added to the nukie planet it's actually MORE expensive to buy the medical bundle than it is to buy the contents. I feel like it has potential though, so here we are.
Carrying a full sized defib with you as a nukie agent is generally not worthwhile, it takes up a large chunk of your bag, you need to drop everything to use it (making you vulnerable to surprise attacks with no way to quickly respond), and requires your allies to NOT use acidifiers, risking their equipment falling into enemy hands.
The interdyne defib solves two of these issues, being half the size, taking up one hand, and being a rather effective melee weapon to use in a pinch.

## Media

https://github.com/user-attachments/assets/fe9df655-3ab2-41ce-91f0-f975c48c53c4

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

<!--
:cl:
- add: Added the interdyne defibrillator, a black-and-red defibrillator that can be used as a melee weapon.
- tweak: The syndicate medical bundle now contains an interdyne defibrillator, a collection of various instant injectors, tourniquets, and several combat kits. The price has been raised to 24 tc.

### CODE REVIEW

**Comment by TurboTrackerss14** [comment]
it should 100% take charge when activated and used as a weapon (use stunprod code ™️ ) because from what I can see its attacks fairly fast and having no chance to resist it is lame. 

Additionally I must say that this entire purchase feels like a noobtrap as any reasonably competent nukie agent should be able to produce these chems. Granted them being autoinjectors helps, but reasonably they would be given out to your squad instead of hoarding yourself and actively working as a medic. 

Them being autoinjectors that would be handed out also does the same issue of "agent being less of a medic" moment as literally everyone on a 4-5 man nukie team would have 1 injector for near every damagetype, allowing agent to return to the loneop-abandonyourteam-dieinmaint-solothegame problem point it was before that issue got tombstoned.

**Comment by thebadman4662** [comment]
> it should 100% take charge when activated and used as a weapon (use stunprod code ™️ ) because from what I can see its attacks fairly fast and having no chance to resist it is lame. 

Stunprods only deal shock damage when turned off, so its probably not the best example here...

      Punct pen

    80 pierce healing
    Stop bleeding

Acid is redundant and possibly even harmful if patient is out of metabolism slots for treatment, 
Pun already stops bleeding. Some saline if anything would be better in the injector, if I recall right nukie syringe version of this right now us 10.5+4.5 mix of Pun/Sal as it can effectively get someone off crit door to perfect health or uncrit anyone gunned down for only 1 grid.

**Comment by Cojoke-dot** [comment]
> > Pun already stops bleeding.
> 
> Pretty sure that's a bug.

Yub
#31137

**Comment by thebadman4662** [comment]
Bug is Lac not treating bleeding too. :trollface:

**Comment by SlamBamActionman** [comment]
> Additionally I must say that this entire purchase feels like a noobtrap as any reasonably competent nukie agent should be able to produce these chems. Granted them being autoinjectors helps, but reasonably they would be given out to your squad instead of hoarding yourself and actively working as a medic.

I agree, but I don't think that's necessarily an issue. Making your own healing will *always* be more TC-efficient for Nukies, not to mention all the free healing kits around on the planet. The benefit of this kit is that you can spread the healing capabilities out across multiple Nukies and you can get access to it immediately; BlitzOps comes to mind. 

> Them being autoinjectors that would be handed out also does the same issue of "agent being less of a medic" moment as literally everyone on a 4-5 man nukie team would have 1 injector for near every damagetype, allowing agent to return to the loneop-abandonyourteam-dieinmaint-solothegame problem point it was before that issue got tombstoned.

Only up to a point, since it's one injector and then it's gone. That being said, I think freeing up the agent is partially the point of the kit; as noted it's not a great use of TC when considering just the medicine, but it does provide the benefit of making the Nukies able to survive more independently. The 24 TC is a steep price to pay for that tactic so I don't think we'll see it being used too often.

To summarize, you either buy this kit if you wanna BlitzOps, you're a LoneOp or if you strategically wanna reduce the pressure on the Agent.

**Comment by thebadman4662** [comment]
Worried about nukies not going for some kind of ssd ops? Nukies being fast is great the way I see it. Even war ops are getting are taking nukies one hour or more to even get to station, anything that makes them get a move on faster and more often is good.

**Comment by KingBandit** [comment]
I feel like in terms of the TC saved it's a little absurd which is where I see this getting trapped up, where the main medical bundle is definitely a good deal, this modification gives you at least 36 TC worth of items not including the defib or additional pens for only 24 TC, realistically unchanged this bundle should and probably will not get accepted unless its closer to 35 or even 40 TC.

More realistically, it would be smart to remove around two of every item, save for maybe the tourniquets? This would reduce its circumstance as a Get-out-of-medic-free-card for the Agent which I've seen some other people point out, it still has some very absurd healing potential and can more than support a nukie time in terms of first aid for the duration of a nukeop. I feel like especially the new medipens while very cool are very overpowered as well and could use another look at, pens that invalidate entire magazines of some guns seem woefully underbalanced, at the least to add en masse to a relatively cheap buy.

I feel like a middle ground of raising the price and removing some of the repeated items would make this a lot more likely to get added in terms of balancing and still keep it as a new common buy for the nuclear team.

**Review by Emisse — APPROVED** [review]
